### ISSUE TITLE
Add API to get information about a specific set of processes

### ISSUE THREAD

**Issue by marcov** [containers/psgo#32]
Hi Valentin, it would be great to have an extra psgo API where you could limit the scope of ProcessInfo to a specified set of pids.

One use case I found is being able to get the the full 64-bit process capabilities bitmask from procfs.
Something like this would probably suffice:

```go
// ProcessInfoByPids is like ProcessInfo but limited to a set of pids
func ProcessInfoByPids(pids []string, descriptors []string) ([][]string, error) {
       aixDescriptors, err := translateDescriptors(descriptors)
       if err != nil {
               return nil, err
       }

       processes, err := process.FromPIDs(pids)
       if err != nil {
               return nil, err
       }

       return processDescriptors(aixDescriptors, processes)
}
```

**Comment by vrothberg** [comment]
@marcov That seems like a nice extension to me. Would you be interested in contributing this API (with some additional tests in `/test/*`)? (Cc @rhatdan)

> One use case I found is being able to get the the full 64-bit process capabilities bitmask from procfs.

Do you mean to get the set of all available capabilities? If so, the manpages seem to be more reliable.

**Comment by vrothberg** [comment]
> Do you mean to get the set of all available capabilities? If so, the manpages seem to be more reliable.

I just saw the other thread in libpod (https://github.com/projectatomic/libpod/pull/1250#issuecomment-412107499). Sorry, I misread that completely. It sounds like a nice use-case :)

**Comment by marcov** [comment]
Cool, it's definitely something I can contribute to :+1:

**Comment by vrothberg** [comment]
Fixed with commit 39bee245d17677ee291b2e234862861853343be2.
